plenario/api/validator.py

```diff
@@ -63,6 +64,8 @@ class Validator(Schema):
     limit = fields.Integer(default=1000)
     offset = fields.Integer(default=0, validate=Range(0))
     resolution = fields.Integer(default=500, validate=Range(0))
+    job = fields.Bool(default=False)
+    all = fields.Bool(default=False)
```

Comment:
I think we should change this field to something that doesn't step on python keywords